### PR TITLE
Added 2 sponsor details

### PR DESCRIPTION
**Pull request by swapnilogale**

### CODE REVIEW

**Comment by swapnilogale** [comment]
@plaindocs - can you review this and approve it for a merge?

**Review by plaindocs — APPROVED** [review]
Looks ok to me, although you should probably delete the `in kind` sponsors page, or include it.

**Comment by swapnilogale** [comment]
@plaindocs - ok, will add this in later. Have a few more updates coming up next few weeks

File: docs/_templates/include/2019/australia-sponsors.html [hunk 1]
```diff
@@ -0,0 +1,52 @@
+<div class="sponsors">
```
Comment (ericholscher) [inline docs/_templates/include/2019/australia-sponsors.html:1]: I don't believe we need these files anymore, we might need to update the site to match the others though.

**Pull request merged**